src/print.h

```diff
@@ -0,0 +1,25 @@
+#ifndef SRC_PRINT_H_
+#define SRC_PRINT_H_
+
+#include "node.h"
+
+#include <stdio.h>
+#include <stdarg.h>
+
+namespace node {
+
+typedef void (*print_func)(const char *format, va_list args);
+typedef void (*print_err_func)(const char *format, va_list args);
+
+NODE_EXTERN print_func SetPrintFunc(print_func func);
+NODE_EXTERN print_err_func SetPrintErrFunc(print_err_func func);
+
+void Print(const char *format, ...);
+void PrintErr(const char *format, ...);
```

Comment:
This is a step back in that we lose the compiler's format string checking this way.  See [this page](https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html) for documentation on gcc's `__attribute__((format(...))` syntax.